patches/chrome-BUILD.gn.patch

```diff
@@ -120,31 +133,42 @@ index 960921acd7ebb0ba99092745e8d30bca76821f2a..534fe8a987d939b2dc24c8cf20d4e950
        "app/chrome_main_mac.h",
        "app/chrome_main_mac.mm",
      ]
-@@ -1226,6 +1268,7 @@ if (is_win) {
+@@ -1226,6 +1274,7 @@ if (is_win) {
  
      public_deps = [
        ":chrome_dll",
 +      "//brave/vendor/brave-extension:copy-extension-localization",
      ]
  
      deps = [
-@@ -1400,6 +1443,7 @@ if (is_win) {
+@@ -1253,6 +1302,10 @@ if (is_win) {
+       "-Wl,-order_file," + rebase_path("app/framework.order", root_build_dir),
+     ]
+ 
++    if (brave_chromium_build && !is_component_build) {
```

Comment:
As I wrote in https://github.com/brave/brave-core/pull/91#discussion_r182969237, it will introduce dependency modification in `//chrome/BUILD.gn`

Reply:
I think `:chrome_framework` needs sparkle link option to build `:chrome_framework_shared_library` target.
In non-component build, `:chrome_framework_shared_library` shared lib links static `//chrome/browser` static library. It needs sparkle linker option because `//chrome/browser` uses sparkle.
The reason why sparkle link option isn't needed by `:chrome_framework` is `:chrome_dll` links sparkle instead.
Sparkle config is propagated to `:chrome_dll`.
But, it is not propagated to `:chrome_framework` in non-component build because `:chrome_dll` has private dependency to `:browser_depencency`.

By using `include name`, rpath options can be removed from linker option.

@bridiver , Please check my comment.

Reply:
After setting `include_name`, test targets such as `brave_browser_test` or `browser_test` can't find sparkle framework when it starts. I tried by adding rpath options to `//brave/test:test_support`, it seems they only use include_name.

Reply:
Ah.. it's typo. I modified `install_name`. In xcode, I changed `Dynamic Library Install Name` property.